docs/Performance.md

```diff
@@ -168,7 +168,21 @@ Access it by selecting Perf Monitor from the Debug menu.
 
 For iOS, Instruments is an invaluable tool, and on Android you should learn to use [`systrace`](docs/performance.html#profiling-android-ui-performance-with-systrace).
 
-You can also use [`react-addons-perf`](https://facebook.github.io/react/docs/perf.html) to get insights into where React is spending time when rendering your components.
+You can also use `RCTRenderingPerf.js` to get insights into where React is spending time when rendering your components. Its behaviour is similar to that of react's [`react-addons-perf`](https://facebook.github.io/react/docs/perf.html)
+
+```js
+import PrefMonitor from 'react-native/Libraries/Performance/RCTRenderingPerf';
```

Comment:
Haha now way... copied it from a snippet from https://github.com/facebook/react-native/issues/12163 and did not pay attention. Fixed.